App.js

```diff
@@ -3,11 +3,15 @@ import { Platform, StatusBar, StyleSheet, View } from 'react-native';
 import { AppLoading, Asset, Font, Icon } from 'expo';
 import AppNavigator from './navigation/AppNavigator';
 
+import { Provider, connect } from 'react-redux';
```

Comment:
minor thing but i don't see "connect" being used here, i think you can remove

Reply:
Just FYI @ernewtoner, in VSCode, unused imports are slightly dimmed.

Reply:
Ah indeed a leftover I thought I was going to use, I noticed that about VSCode, very useful!